src/admin/resolvers/queries/ApprovedItem.auth.integration.ts

```diff
@@ -0,0 +1,264 @@
+import { expect } from 'chai';
+import { CuratedStatus } from '@prisma/client';
+import { ACCESS_DENIED_ERROR, MozillaAccessGroup } from '../../../shared/types';
+import { db } from '../../../test/admin-server';
+import {
+  clearDb,
+  createApprovedItemHelper,
+  getServerWithMockedHeaders,
+} from '../../../test/helpers';
+import {
+  GET_APPROVED_ITEMS,
+  GET_APPROVED_ITEM_BY_URL,
+} from './sample-queries.gql';
+
+describe('queries: ApprovedCuratedCorpusItem - authentication', () => {
+  beforeAll(async () => {
+    // clear out db to start fresh
+    await clearDb(db);
+  });
+
+  afterAll(async () => {
+    await db.$disconnect();
+  });
+
+  describe('getApprovedCuratedCorpusItems query', () => {
+    beforeAll(async () => {
+      // Create some items
+      const stories = [
+        {
+          title: 'How To Win Friends And Influence People with GraphQL',
+          language: 'en',
+          status: CuratedStatus.RECOMMENDATION,
+          topic: 'FOOD',
+        },
+        {
+          title: 'What Zombies Can Teach You About GraphQL',
+          language: 'en',
+          status: CuratedStatus.RECOMMENDATION,
+          url: 'https://www.sample-domain/what-zombies-can-teach-you-graphql',
+          topic: 'TECHNOLOGY',
+        },
+        {
+          title: 'How To Make Your Product Stand Out With GraphQL',
+          language: 'en',
+          status: CuratedStatus.RECOMMENDATION,
+          topic: 'TECHNOLOGY',
+        },
+      ];
+
+      // insert test stories into the db
+      for (const story of stories) {
+        await createApprovedItemHelper(db, story);
+      }
+    });
```

Comment:
It looks like the newly added files (for the `getApprovedCuratedCorpusItemByUrl` query) are relying on the data seeded in another `describe()` block - not the top-level one but a block on the same level that tests auth checks for `getApprovedCuratedCorpusItems`. 

Would it make sense to move it up to the top-level `describe() -> beforeAll()` so that when we come back to these tests in a while there is no confusion about where the data for the first query is coming from?